frontend/src/services/store.ts

```diff
@@ -156,6 +162,7 @@ let store = withEffects(createStore<Actions>({
   counties: [],
   error: null,
   success: null,
+  alert: isMobile ? 'Encompass is not optimized for mobile devices yet. Please visit using a desktop browser for better performance and usability.' : null,
```

Comment:
I think maybe this should be in App.ts and not here.
Having logic in the store seems a bit weird. I would have it null here, but have the webgl / mobile logic testing in App. What do you think?

Reply:
If it's in app.ts it's hard to have it set to alert to start then have it close after.

Reply:
Ahh :( Why? Is it not similar to what we had before?

Reply:
no we had it as an alert before, which doesn't involve the store at all, and lived outside the app component. For this, because it has the store, it needs to be inside the component, and will re-render any time things change so it's not possible to tell it that it's the first time it's being loaded.